### PR TITLE
Rep command

### PR DESCRIPTION
**Pull request by Doges**
hmmm bolinha de gorfe

### CODE REVIEW

File: src/commands/social/rep.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+const { CommandStructures, SwitchbladeEmbed, Constants } = require('../../')
+const { Command, CommandParameters, UserParameter } = CommandStructures
+const moment = require('moment')
+
+const DAILY_INTERVAL = 24 * 60 * 60 * 1000 // 1 day
```
Comment (bolsomito) [inline src/commands/social/rep.js]: renaem

File: src/commands/social/rep.js [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+const DAILY_INTERVAL = 24 * 60 * 60 * 1000 // 1 day
+
+module.exports = class Rep extends Command {
+  constructor (client) {
+    super(client)
+    this.name = 'rep'
+
+    this.parameters = new CommandParameters(this,
+      new UserParameter({missingError: 'commands:test.nomention', acceptBot: true, test: true})
```
Comment (bolsomito) [inline src/commands/social/rep.js]: huh?

File: src/locales/en-US/commands.json [hunk 3]
```diff
@@ -400,5 +400,11 @@
     "commandDescription": "Set a background color to your profile",
     "commandUsage": "<#hexcode>",
     "changedSuccessfully": "Profile color changed successfully to {{hexcode}}."
+  },
+  "rep": {
```
Comment (bolsomito) [inline src/locales/en-US/commands.json]: where's commandDescription and commandUsage? :thinking:

**Comment by pedrofracassi** [comment]
image

**Comment by Doges** [comment]
@bolsomito also, tu esqueceu de falar sobre o schema, me lembrei agora

**Comment by bolsomito** [comment]
@Doges well played

**Comment by Doges** [comment]
shit

**Comment by pedrofracassi** [comment]
I think it should be shown on the profile

**Comment by Doges** [comment]
@pedrofracassi don't fuck with my pr, open an issue l8

**Pull request merged**